modules/Cockpit/assets/components/field-text.tag

```diff
@@ -2,7 +2,7 @@
 
     <style>
 
-        [ref="input"] {
+        [type=text] {
```

Comment:
could you please leave the `[ref="input"]` here. so the change should look like this:

`[ref="input"][type="text"]`

Reply:
OK, I changed it.